components/confirm_modal.jsx

```diff
@@ -85,19 +85,19 @@ export default class ConfirmModal extends React.Component {
 
     componentDidMount() {
         if (this.props.show) {
-            document.addEventListener('keypress', this.handleKeypress);
+            document.addEventListener('keydown', this.handleKeypress);
```

Comment:
This change seems orthogonal to the new feature -- what's the context?

Reply:
as far I remember, @saturninoabril and I checked that and to use the enter key across modal we made this change because the enter key was not captured by the modal and was captured by another component. 
let me know if you want to change that.

Reply:
That's right.  It's weird that pressing `enter` key works on `keydown` and not on `keypress` in this component.